cli/tests/pcluster/config/test_section_aws.py

```diff
@@ -14,6 +14,26 @@
 from pcluster.config.mappings import AWS
 
 
+@pytest.fixture()
+def default_region_mock():
```

Comment:
why not mocking boto3.session.Session().region_name rather than replacing the content of mappings?